app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -46,7 +45,6 @@ import kotlinx.coroutines.Dispatchers.IO
 import kotlinx.coroutines.Dispatchers.Main
 import kotlinx.coroutines.MainScope
 import kotlinx.coroutines.delay
-import kotlinx.coroutines.flow.collect
```

Comment:
I'm not seeing `Flow.collect` removed anywhere in this changelog. I think we should probably revert this change to be on the safe side, unless I'm missing something.

Reply:
> This is looking great so far, thanks again for your contribution!
> 
> There are a couple more candidates for removal. They could be done as part of this patch, or they could be done as separate follow-ups after we land this patch. I will leave that up to you. No pressure if you are not interested in them, just let me know slightly_smiling_face
> 
> 1. There is a function in `WallpaperManager` called [switchToNextWallpaper](https://searchfox.org/mozilla-mobile/rev/712f60261a3cef9495814eb49148d0692c350e42/fenix/app/src/main/java/org/mozilla/fenix/wallpapers/WallpaperManager.kt#74) that was called when the logo was clicked. That should be removed, and IMO would make sense to do as part of this patch.
> 2. There is another user preference for whether the toggle works at all called [wallpapersSwitchedByLogoTap](https://searchfox.org/mozilla-mobile/rev/712f60261a3cef9495814eb49148d0692c350e42/fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt#193). This preference and the code that uses it also need to be removed. This one would make sense as part of this patch, but I think could also be done in a follow-up.
> 
> So please let me know if you would like to add either or both of those things in this patch or in follow-ups.

I have done both the changes. 
Actually i had removed switchToNextWallpaper in the first commit, but i added it back during resolving conflicts because i saw some changes related to its use, so thought will remove it once you confirm we're removing it.
Thanks for your review.

Reply:
> I'm not seeing `Flow.collect` removed anywhere in this changelog. I think we should probably revert this change to be on the safe side, unless I'm missing something.

Yes. reverted it.